### PR TITLE
adds checkboxes to code review section of PR template

### PR DESCRIPTION
**Pull request by andy5995**
[skip ci]

# Description

Adds checkboxes to code review section of PR template

# Code review items

- [x] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
 
-# Code review items
-
-- Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
-- Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
-- Tested: Unit and/or e2e where appropriate
-- Internationalised: All user facing text
-- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.
+# Code review checklist
+<!-- Remove or comment out any items that do not apply to this PR; in the remaining boxes, replace the [ ] with [x]. -->
```
Comment (craig-landry) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: Thanks @andy5995, this looks like a nice feature to take advantage of. If I understand correctly, the html comment syntax here will be seen when viewing it in an editor, but not when displayed in a browser (viewing a PR on GitHub). Is that correct? If so, this looks good to me. If the characters will render literally in the browser though, I'd like it to be modified to just be a regular line of text.

Comment (andy5995) [reply .github/PULL_REQUEST_TEMPLATE.md:8]: @craig-landry I'm glad you like it. Yes, the comment syntax works here. To preview this, you can click on the "files changed" tab of the PR -> click the "more"(3 vertical dots) icon on the right -> view file.

**Review by craig-landry — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (craig-landry) [reply .github/PULL_REQUEST_TEMPLATE.md:8]: Nice, good stuff! Thanks @andy5995.

**Comment by andy5995** [comment]
I think this is ready for merging ;)

**Comment by garethbowen** [comment]
@craig-landry I think we decided to merge this?

**Comment by craig-landry** [comment]
> @craig-landry I think we decided to merge this?

Thanks, yup. Thanks for the prompt. It's currently stuck on CI (that won't happen). Would you please force merge it @garethbowen ?

**Comment by garethbowen** [comment]
Done!

**Pull request merged**